src/components/subscriptions/licenses/LicenseManagementModals/LicenseManagementRevokeModal.jsx

```diff
@@ -71,42 +66,50 @@ const LicenseManagementRevokeModal = ({
   usersToRevoke,
   revokeAllUsers,
   totalToRevoke,
+  activeFilters,
 }) => {
   const [requestState, setRequestState, initialRequestState] = useRequestState(isOpen);
 
-  const buttonLabels = generateRevokeModalSubmitLabel(
-    revokeAllUsers,
-    usersToRevoke.length,
-    totalToRevoke,
-  );
+  const buttonLabels = generateRevokeModalSubmitLabel(totalToRevoke);
 
-  const title = `Revoke License${revokeAllUsers || usersToRevoke.length > 1 ? 's' : ''}`;
+  const title = `Revoke License${revokeAllUsers || totalToRevoke > 1 ? 's' : ''}`;
 
   const isExpired = moment().isAfter(subscription.expirationDate);
 
-  const handleSubmit = () => {
+  const handleSubmit = async () => {
```

Comment:
nit: I wonder if this should use `useCallback` to only create the function definition when it needs to versus on every component re-render 🤔